cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,6 +19,26 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
+  get countryField() {
+    return cy.get('#country');
+  }
+
+  get cityField() {
+    return cy.get('#city');
+  }
+
+  get creditCardField() {
+    return cy.get('#card');
+  }
+
+  get monthField() {
+    return cy.get('#month');
+  }
+
+  get yearField() {
+    return cy.get('#year');
+  }
+
   typeEmail(email) {
     this.emailField.type(email, { force: true });
```

Comment:
Why do we need `force` here?
If the method with this `emailField` - try another CSS class of selector

Reply:
this part of code isn't mine, but I can suppose, that  {forse: true} statement is used for hidden elements or solving problems with elements beyond of the cypress browser window